packages/bot-web-ui/src/components/bot-notification-messages/bot-notification-messages.tsx

```diff
@@ -18,8 +18,8 @@ const BotNotificationMessages = ({
 }: TBotNotificationMessagesProps) => (
     <div
         className={classNames('notifications-container', {
-            'notifications-container--panel-open':
-                active_tab === 0 ? is_info_panel_visible : (active_tab === 1 || active_tab === 2) && is_drawer_open,
+            'notifications-container__dashboard': active_tab === 0 && is_info_panel_visible,
+            'notifications-container--panel-open': [1, 2, 3].includes(active_tab) && is_drawer_open,
```

Comment:
Can we have this `[1, 2, 3]` exported in one place? I think this is used in multiple places. In the future if we are adding a new tab it will be easier for us to update.

Reply:
done ✅